templates/commit-snippet-2

```diff
@@ -1,7 +1,8 @@
 # Code for checking consistency of @@EBUILD@@
 pushd @@CP@@ > /dev/null
 
-repoman -d full || exit 1
-repoman -m "@@CP@@: ${DESCR}" commit || exit 1
+pkgcheck scan || exit 1
```

Comment:
Not strictly needed, `pkgdev commit` does a scan anyway

Reply:
`pkgdev commit` only scans when configured to, so this is good/needed